internal/controller/entra/custom_mail.go

```diff
@@ -48,10 +84,41 @@ func (c *Controller) SendCustomInviteNotification(ctx context.Context, email str
 	userID := config.EntraMailUserPrincipal()
 
 	// use the mail client to send via the mailbox
-	err := c.mailClient.Users().ByUserId(userID).SendMail().Post(ctx, requestBody, nil)
+	err = c.mailClient.Users().ByUserId(userID).SendMail().Post(ctx, requestBody, nil)
 	if err != nil {
 		return types.NewErrServerError(err)
 	}
 
 	return nil
+
+}
+
+func (c *Controller) SendCustomInviteNotification(ctx context.Context, email string, sponsor types.Sponsor) error {
+	// use graph to send email saying so-and-so has invited you to the portal
+
+	content := ""
+	if sponsor.ChosenName != "" {
+		content = "You have been invited to join the UCL ARC Services Portal by " + string(sponsor.ChosenName)
+	} else {
+		content = "You have been invited to join the UCL ARC Services Portal by " + string(sponsor.Username)
+	}
+
+	return c.createCustomEmail(ctx, "Notification: You have been invited to the UCL ARC Services Portal", []string{email}, content)
+}
+
+func (c *Controller) SendCustomStudyReviewNotification(ctx context.Context, email string, review openapi.StudyReview) error {
+	// email to notify IAO + IAA when study has been reviewed
+	// todo: set up usage of this
+
+	content := ""
+	subject := ""
+	if review.Status == openapi.Approved {
+		subject = "Notification: Your study has been approved!"
+		content = "Your study has been approved! You will be notified when you have any contracts or assets approaching expiry."
+	} else {
```

Comment:
I'd switch on rejected, then not send if the status is anything else? seems safer

Reply:
https://github.com/ucl-arc-tre/portal/pull/336/commits/b7d290d9aa1a38ac3f32af015426e6f0a357b33c ? or something different?

Reply:
is there a difference between setting it first vs using `default`? https://github.com/ucl-arc-tre/portal/pull/336/commits/75a455b1b867557a5552a159721841e3c35a6b48

Reply:
nah – all good 🚀